controller.go

```diff
@@ -119,7 +120,15 @@ type driverData struct {
 	capability driverapi.Capability
 }
 
+type ipamData struct {
+	config    ipamapi.Config
+	allocator ipamapi.Allocator
+	// default address spaces are provided by ipam driver at registration time
+	defaultLocalAddressSpace, defaultGlobalAddressSpace string
```

Comment:
Let me give you some context (sorry if redundant). 
- The default address space is needed in case UI does not passes it (which will be always the case for first releases). We want the ipam driver to let us know which is the default addr space string.
- We need two default address spaces, one for local scoped networks (like for ex. bridge) and one to be used for global scoped networks (like for ex. overlay). Because on a docker host both kind of networks can co-exists.